### ISSUE TITLE
Maybe this  is old source

### ISSUE THREAD

**Issue by lcx19950201** [afulsamet/KateBot-Remastered#9]
I don't think this is the latest source, because the latest source can automatically obtain the player's address in memory. This doesn't work, I think it doesn't get the character address in memory, and I can't understand the new config.ini, which doesn't conform to the script rules of kateBot, maybe you modified config.cpp. If you have a new source ,look forward to your reply

**Comment by real-Shigure** [comment]
Yes,the real katebot can find release offset automatically. This is why it hardly needs updating. In fact, I am also very interested in the principle of automatic offset search.
